parser.go

```diff
@@ -156,25 +174,28 @@ Returns a identifier with which the handler can be removed via UnregisterNetMess
 
 See also: RegisterEventHandler()
 */
-func (p *Parser) RegisterNetMessageHandler(handler interface{}) dp.HandlerIdentifier {
-	return p.msgDispatcher.RegisterHandler(handler)
+
+type HandlerIdentifier *int
```

Comment:
It's the same issue as discussed in #40 :slightly_smiling_face:

Reply:
Ahhh right, I see 🙂 - I've actually been thinking of merging the `vgo` branch any time now since it's actually quite usable with go 1.11 (when outside the `GOPATH` or with `GO111MODULE=on`).

So if you are able to switch to vgo instead maybe that would be the preferred route?

Of course this will also require some docs from my end on how to use the lib like this since not everyone is used to modules yet.

Reply:
Yep, moving to `vgo` definitely could be a solution! That would be fine by me :slightly_smiling_face:

Reply:
I've reverted the changes accordingly.